Formula/b/blink.rb

```diff
@@ -1,8 +1,9 @@
 class Blink < Formula
   desc "Tiniest x86-64-linux emulator"
   homepage "https://github.com/jart/blink"
-  url "https://github.com/jart/blink/releases/download/1.0.0/blink-1.0.0.tar.gz"
-  sha256 "09ffc3cdb57449111510bbf2f552b3923d82a983ef032ee819c07f5da924c3a6"
+  url "https://github.com/jart/blink/releases/download/1.1.0/blink-1.1.tar.gz"
+  version "1.1.0"
```

Comment:
Added `version` here because while x.y.z style tags are used, release asset is just x.y. Binary still reports version in the x.y.z style so it seems correct to keep that versioning scheme even if releases drop the .z.

```
blink -v

Blink Virtual Machine 1.1.0 (Sun Jan 21 18:50:33 UTC 2024)
Copyright (c) 2023 Justine Alexandra Roberts Tunney
Blink comes with absolutely NO WARRANTY of any kind.
You may redistribute copies of Blink under the ISC License.
For more information, see the file named LICENSE.
Toolchain: Apple clang version 15.0.0 (clang-1500.1.0.2.5)
Revision: # 
Config: ./configure MODE= --prefix=/opt/homebrew/Cellar/blink/1.1.0 --enable-vfs
```

Reply:
@timsutton can you submit an upstream question about it? Thanks!

Reply:
The release is advertised as:

<img width="521" alt="Capture d’écran 2024-01-21 à 20 24 29" src="https://github.com/Homebrew/homebrew-core/assets/1980544/b05ad3a0-56ed-44e9-9eaf-2ca9f11312b6">

Reply:
Yep! Done here: https://github.com/jart/blink/issues/164

Reply:
LMK if it's preferable to include that in the Formula .rb file as a comment and I can re-update my commit to include that.

Reply:
I think it is better to have as 1.1.0, especially when that is part of the version output :)